vms/platformvm/block/builder/helpers_test.go

```diff
@@ -88,17 +73,22 @@ type environment struct {
 }
 
 func newEnvironment(t *testing.T) *environment {
-	require := require.New(t)
+	r := require.New(t)
+
+	var (
+		fork     = ts.DurangoFork
+		forkTime = ts.ValidateStartTime.Add(-2 * time.Second)
```

Comment:
different packages still have different fork times set. We'll fix this in next PR https://github.com/ava-labs/avalanchego/pull/2169